docs/content/reference/resources/memory.mdx

```diff
@@ -63,38 +73,61 @@ Memory is implemented as a simple HTTP server:
 |--------|----------|-------------|
 | POST | `/messages` | Store multiple messages |
 | GET | `/messages` | Retrieve messages with optional filtering |
-| GET | `/sessions` | List all session IDs |
+| POST | `/conversations/init` | Initialize or get a conversation ID |
+| GET | `/conversations` | List all conversation IDs |
+| DELETE | `/conversations/{conversationId}` | Delete a specific conversation |
 | GET | `/health` | Health check |
 
 ### Store Messages
 
 **POST** `/messages`
 
-Request body includes session/query context and message array:
+Request body includes conversation/query context and message array:
 
 ```json
 {
-  "session_id": "uuid-string",
-  "query_id": "query-uuid", 
+  "conversation_id": "conv-abc123",
+  "query_id": "query-uuid",
   "messages": [
     {
       "role": "user",
       "content": "What is the weather like?"
     },
     {
-      "role": "assistant", 
+      "role": "assistant",
       "content": "I don't have access to real-time weather data."
     }
   ]
 }
 ```
 
+If `conversation_id` is not provided, a new one will be generated and returned in the response.
+
+### Initialize Conversation
+
+**POST** `/conversations/init`
```

Comment:
This is the only thing that doesn't quite make sense to me here, is there a more restful way? Are we potentially just `POST`-ing into conversations? This might be easier to talk over